db/migrate/20230509122013_add_post_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :post, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Good job so far but the table `likes` has also a reference to the users' table that will create `author_id` and a corresponding index, please kindly add it 🙏🏾